pkgs/shells/fish/default.nix

```diff
@@ -306,40 +291,6 @@ let
     };
 
     passthru = {
-      shellPath = "/bin/fish";
-      tests = {
-        nixos = lib.optionalAttrs stdenv.isLinux nixosTests.fish;
-
-        # Test the fish_config tool by checking the generated splash page.
-        # Since the webserver requires a port to run, it is not started.
-        fishConfig =
-          let fishScript = writeText "test.fish" ''
```

Comment:
We could fix the test by adding python to `PATH`.

Reply:
I fixed it by adding it to the `buildInputs` in the `runCommand`. Is it OK for you?